testing/api.spec.js

```diff
@@ -0,0 +1,54 @@
+const database = require('../database/index.js');
+const axios = require('axios');
+const mongoose = require('mongoose');
+// import Overview from '../client/overview';
+
+beforeAll(async () => {
+  await mongoose.connect('mongodb://127.0.0.1/overview', { useNewUrlParser: true, useCreateIndex: true, useFindAndModify: false })
+});
+
+afterAll(async done => {
+  done();
+})
+
+test('checks database general look up function', async () => {
+  await mongoose.connect('mongodb://127.0.0.1/overview', { useNewUrlParser: true, useCreateIndex: true, useFindAndModify: false });
```

Comment:
How come we need to connect to DB twice? You answered this.. but this was the only comment I had lol